crates/swc_css_ast/src/base.rs

```diff
@@ -100,12 +101,39 @@ impl Take for SimpleBlock {
     }
 }
 
+#[ast_node]
+#[derive(Eq, Hash, Is, EqIgnoreSpan)]
+pub enum FunctionName {
```

Comment:
Why is this required?

Reply:
@kdy1 Yeah, it reduce code, we have the same for `AtRule` and `Declaration`, so we will have less code